pallets/author-mapping/src/migrations.rs

```diff
@@ -60,6 +60,8 @@ impl<T: Config> OnRuntimeUpgrade for AddAccountIdToNimbusLookup<T> {
 	#[cfg(feature = "try-runtime")]
 	fn pre_upgrade() -> Result<(), &'static str> {
 		use frame_support::traits::OnRuntimeUpgradeHelpersExt;
+		use sp_std::vec::Vec;
```

Comment:
Is this needed ?

Reply:
It was needed for `try-runtime` I believe, although that doesn't currently build (I think we need https://github.com/PureStake/nimbus/pull/62)